code/controllers/subsystem.dm

```diff
@@ -1,39 +1,88 @@
-
+//**
```

Comment:
single /

Reply:
```suggestion
/**
```

Adding so i can just hit a button to commit the change